src/material/tabs/tab-group.html

```diff
@@ -1,10 +1,10 @@
 <mat-tab-header #tabHeader
-               [selectedIndex]="selectedIndex"
+               [selectedIndex]="selectedIndex || 0"
                [disableRipple]="disableRipple"
                (indexFocused)="_focusChanged($event)"
                (selectFocusedIndex)="selectedIndex = $event">
   <div class="mat-tab-label" role="tab" matTabLabelWrapper mat-ripple cdkMonitorElementFocus
-       *ngFor="let tab of _tabs; let i = index"
+       *ngFor="let tab of _tabs.toArray(); let i = index"
```

Comment:
This isn't very efficient, because `toArray` returns a new array on each change detection. If `ngFor` requires an array, we should have a different property that we only re-assign when the tabs change.

Reply:
What should we call the field? `_tabsArray`? `_tabsIterable`?

Reply:
I'd say `_tabsArray`.

Reply:
Done for Tabs and Stepper.